plugins/modules/swim_workflow_manager.py

```diff
@@ -565,23 +617,54 @@ def site_exists(self, site_name):
         site_exists = False
         site_id = None
         response = None
+
         try:
-            response = self.dnac._exec(
-                family="sites",
-                function='get_site',
-                op_modifies=True,
-                params={"name": site_name},
-            )
+            if self.dnac_version <= self.version_2_3_5_3:
+                response = self.dnac._exec(
+                    family="sites",
+                    function='get_site',
+                    op_modifies=True,
+                    params={"name": site_name},
+                )
```

Comment:
I can see that this API is present in older version as well, for example: it's present in v2_3_3_0.

Reply:
Yes, we verified that this API is available in the older version, including v2_3_3_0. We've implemented a condition in the code:

if self.dnac_version <= self.version_2_3_5_3:

This converts dnac_version into an integer, and if the version is less than 2353, the functionality will be executed. This approach ensures that the functionality works for all versions below 2.3.5.3.